src/Symfony/Component/Lock/Store/MongoDbStore.php

```diff
@@ -88,7 +92,7 @@ class MongoDbStore implements PersistingStoreInterface
      * readPreference is primary for all queries.
      * @see https://docs.mongodb.com/manual/applications/replication/
      */
-    public function __construct(Collection|Client|string $mongo, array $options = [], float $initialTtl = 300.0)
+    public function __construct(Collection|Client|Manager|string $mongo, array $options = [], float $initialTtl = 300.0)
```

Comment:
I haven't added `MongoDB\Database` as intermediary accepted type. There is no real use-case but that would be easy to implement.
The main motivation would be to prevent unexpected error if someone pass an instance of `MongoDB\Database` (Stringable), it would be be casted to `string` and used as DSN.

Reply:
Added support for `MongoDB\Database`